.github/autolabeler.yml

```diff
@@ -0,0 +1,83 @@
+
+# The following paragraph was taken from the Apache Spark project.
+#   see: https://github.com/apache/spark/blob/master/.github/autolabeler.yml
+# Bot page: https://github.com/apps/probot-autolabeler
+# The matching patterns follow the .gitignore spec.
+# See: https://git-scm.com/docs/gitignore#_pattern_format
+# Also, note that the plugin uses 'ignore' package. See also
+# https://github.com/kaelzhang/node-ignore
+
+INFRA:
+  - ".baseline/"
+  - ".github/"
+  - "/dev/"
+  - ".asf.yaml"
+  - ".gitattributes"
+  - ".gitignore"
+  - "baseline.gradle"
+  - "deploy.gradle"
+  - "jitpack.yml"
+  - "site/mkdocs.yml"
+  - "travis.yml"
+BUILD:
+  - "/dev/"
+  - "/project/"
+  - "/assembly/"
+  - "*pom.xml"
```

Comment:
This project doesn't use maven. Should we remove this? And what about the files that are also in .gitignore?

Reply:
Woops this was left over from the version I was using in the spark project. I'll remove the maven references etc.

As for the fils in the .gitignore, I can remove those too if we'd like.

Reply:
Unless I'm mistaken, any file in the .gitignore shouldn't be able to make its way into a PR, so that should not be a problem (unless I'm not understanding the question correctly). 

The .gitignore file itself however is labeled as infra. Let me know if this does not cover your concern.